src/seeding.jl

```diff
@@ -40,22 +40,21 @@ initseeds_by_costs(algname::Symbol, costs::AbstractMatrix{<:Real}, k::Integer) =
 initseeds(iseeds::Vector{Int}, X::AbstractMatrix{<:Real}, k::Integer) = iseeds
 initseeds_by_costs(iseeds::Vector{Int}, costs::AbstractMatrix{<:Real}, k::Integer) = iseeds
 
-function copyseeds!(S::AbstractMatrix{T}, X::AbstractMatrix{T},
-                    iseeds::AbstractVector) where T<:Real
-    d = size(X, 1)
-    n = size(X, 2)
+function copyseeds!(S::Matrix{<:AbstractFloat}, X::AbstractMatrix{<:Real},
+                    iseeds::AbstractVector)
+    d, n = size(X)
     k = length(iseeds)
-    (size(X,1) == d && size(S) == (d, k)) ||
-        throw(DimensionMismatch("Inconsistent array dimensions."))
-
+    size(S) == (d, k) || throw(DimensionMismatch("Inconsistent array " *
```

Comment:
There's two spaces in the message after your change. I think it's fine to keep it on the single line (~80 chars)

```suggestion
    size(S) == (d, k) || throw(DimensionMismatch("Inconsistent array dimensions."))
```